src/components/home/PanelResult.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <v-card outlined>
+    <v-card-text>
+      <v-data-table
+        :headers="tableHeaders"
+        :items="getPanelResult()"
+        item-key="name">
+        <template v-slot:top>
+          <v-toolbar flat>
+            <v-toolbar-title>{{ $t('panel-result.result.name') }}<span v-if="userGenes.length > 0">({{ $t('panel-result.result.title') }}: {{ userGenes.length }})</span></v-toolbar-title>
+          </v-toolbar>
+          <v-dialog v-model="showDialog" persistent max-width="800px">
+            <v-card>
+              <v-card-title>
+                <span class="text-h5" v-if="geneType === 'genesInPanel'">{{ $t('panel-result.dialog.title-genes-in-panel') }}: {{ panelName }}</span>
+                <span class="text-h5" v-else>{{ $t('panel-result.dialog.title-genes-not-in-panel') }}: {{ panelName }}</span>
+              </v-card-title>
+              <v-card-text>
+                <v-virtual-scroll :items="genes" height="400" item-height="64">
+                  <template v-slot:default="{ item }">
+                    <v-list-item :key="item">
+                      <v-list-item-content> <v-list-item-title> {{ item }} </v-list-item-title> </v-list-item-content>
+                    </v-list-item>
+                    <v-divider></v-divider>
+                  </template>
+                </v-virtual-scroll>
+              </v-card-text>
+              <v-card-actions>
+                <v-spacer></v-spacer>
+                <v-btn color="green darken-1" text @click="showDialog = false"> {{ $t('panel-result.dialog.button.close') }} </v-btn>
+                <v-btn color="green darken-1" text @click="downloadGenes(genes)">
+                  {{ $t('panel-result.dialog.button.save') }}
+                </v-btn>
+              </v-card-actions>
+            </v-card>
+          </v-dialog>
+        </template>
+        <template v-slot:item.countGenesInPanel="{ item }">
+          <v-tooltip bottom>
+            <template v-slot:activator="{ on }">
+              <v-chip v-on="on" @click.stop="openDialog(item, 'genesInPanel')" class="ma-2" color="primary" text-color="white"> {{ item.countGenesInPanel }}</v-chip>
+            </template>
+            <span>{{ $t('panel-result.chip.show-genes') }}</span>
+          </v-tooltip>
+        </template>
+        <template v-slot:item.countGenesNotInPanel="{ item }">
```

Comment:
Replace all item v-slots with the square bracket format:  v-slot:[`item.countGenesNotInPanel`]="{ item }"

Reply:
done